src/main/java/org/gridsuite/modification/server/entities/equipment/deletion/EquipmentDeletionEntity.java

```diff
@@ -27,6 +30,17 @@ public class EquipmentDeletionEntity extends EquipmentModificationEntity {
     @Column(name = "equipmentType")
     private String equipmentType;
 
+    @Column(name = "hvdcWithLCC", columnDefinition = "boolean default false")
+    private boolean hvdcWithLCC;
+
+    @ElementCollection
+    @CollectionTable(name = "shuntCompensatorsSide1")
+    private List<ShuntCompensatorSelectionEmbeddable> shuntCompensatorsSide1;
```

Comment:
I think we have a pb of design
This entity is a general class and we add some specific informations for hvdcline

Reply:
As discussed, both DTO and entity will have a generic and optional member "specificData"